feat!: add default alerts configuration

adugast opened the pull request.

Commit:
feat: add default alerts configuration

Commit:
fix: correct deploymentId description

Commit:
fix: set deploymentId in github workflow

Commit:
fix: add fail if slack-token is empty

Commit:
fix: change condition in secret-grafana-slack

dimgogos committed:
fix: disable manage alerts for Loki, deploy rollout on slack token change

dimgogos committed:
fix: set alerts to paused if alerting is disabled

dimgogos committed:
fix: linting issue

Commit:
fix: set notifications repeat interval to 1h

Commit:
fix: create validation file and fix typos

dimgogos committed:
fix: update logic for empty/non-existing slack token

Wielewout reviewed (commented):
I'll push the changes in a new commit (also attempting to fix the ct install test failure).

Wielewout committed:
refactor: simplify existing slack token logic

Wielewout committed:
ci: fix indentation

Yaml requires with the `>` operator that everything is indented the same.
Otherwise newlines are not replaced by a space at these locations.
